src/views/EditSimple.vue

```diff
@@ -334,6 +358,14 @@ export default {
 		window.removeEventListener('keydown', this.keyboardDuplicateEvent)
 	},
 	methods: {
+		closePopover() {
+			this.showMask = false
+		},
+		cssVariables() {
+			return {
+				'--slideshow-duration': '300ms',
```

Comment:
is this used?

Reply:
no, leftover of hasPreview and hasNext button, from NcModal that we dont need here :). Removed